api_app/urls.py

```diff
@@ -30,6 +31,7 @@
     path("ask_analysis_result", ask_analysis_result),
     path("get_analyzer_configs", get_analyzer_configs),
     path("download_sample", download_sample),
+    path("job/kill/", kill_running_job),
```

Comment:
Change the URL format to `/api/kill_analysis?job_id`

Reply:
This can be resolved as well.